cglims/api.py

```diff
@@ -34,6 +35,13 @@ def sample(self, lims_id, is_cgid=False):
                 return None
         else:
             lims_sample = Sample(self, id=lims_id)
+
+        # add dynamic property
+        if lims_sample.get('tissue_type') != 'tumour':
+            apptag = ApplicationTag(lims_sample.udf['Sequencing Analysis'])
+            pipeline = 'mip' if apptag.is_human else 'mwgs'
+        lims_sample.__dict__['pipeline'] = pipeline
```

Comment:
why are you accessing a hidden attr? why not just set the "pipeline" attr directly?